### PR TITLE
Add article: WASM based Dimension Reduction

### PR DESCRIPTION
**Pull request by J16N**
This pull request adds the article: WASM based Dimension Reduction, describing my QSOC 2024 journey.

### CODE REVIEW

File: qdrant-landing/content/articles/dimension-reduction-qsoc.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+
+## Introduction
+
+Hello, everyone! I'm Jishan Bhattacharya, and I had the incredible opportunity to intern at Qdrant this summer as part of the Qdrant Summer of Code 2024. Under the mentorship of the brilliant Andrey Vasnetsov, I dived into the world of performance optimization, focusing on enhancing vector visualization using WebAssembly (WASM). In this article, I'll share the insights, challenges, and accomplishments from my journey — one filled with learning, experimentation, and plenty of coding adventures.
+
+
+## Project Overview
+
+Hello, everyone! I'm Jishan Bhattacharya, and I had the incredible opportunity to intern at Qdrant this summer as part of the Qdrant Summer of Code 2024. Under the mentorship of the brilliant Andrey Vasnetsov, I dived into the world of performance optimization, focusing on enhancing vector visualization using WebAssembly (WASM). In this article, I'll share the insights, challenges, and accomplishments from my journey — one filled with learning, experimentation, and plenty of coding adventures.
```
Comment (generall) [inline qdrant-landing/content/articles/dimension-reduction-qsoc.md:32]: Duplicated parts?

Comment (J16N) [reply qdrant-landing/content/articles/dimension-reduction-qsoc.md]: Fixed it.

File: qdrant-landing/content/articles/dimension-reduction-qsoc.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+I introduced a "rendering-on-demand" approach, where the main thread would signal the worker thread when it was ready to render the next result. This created smoother, more responsive animations.
+
+{{< figure 
+    src="/articles_data/dimension-reduction-qsoc/curr_arch.png" 
+    caption="Current architecture with rendering-on-demand"
+    alt="Image showing the current architecture of the frontend with rendering-on-demand approach"
+>}}
+
+With these optimizations in place, the final step was wrapping up the project by creating a Node.js package. This package exposed the necessary interfaces to accept the distance matrix, perform calculations, and return the results, making the solution easy to integrate into various projects.
```
[the remark this answers is not in the document]
Comment (J16N) [reply qdrant-landing/content/articles/dimension-reduction-qsoc.md]: Added it.

File: qdrant-landing/content/articles/dimension-reduction-qsoc.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    - vector similarity
+    - tsne
+    - qdrant data visualization
+---
+
+
+
+## Introduction
+
+Hello, everyone! I'm Jishan Bhattacharya, and I had the incredible opportunity to intern at Qdrant this summer as part of the Qdrant Summer of Code 2024. Under the mentorship of the brilliant [Andrey Vasnetsov](https://www.linkedin.com/in/andrey-vasnetsov-75268897/), I dived into the world of performance optimization, focusing on enhancing vector visualization using WebAssembly (WASM). In this article, I'll share the insights, challenges, and accomplishments from my journey — one filled with learning, experimentation, and plenty of coding adventures.
```
Comment (generall) [inline qdrant-landing/content/articles/dimension-reduction-qsoc.md:27]:
```suggestion
Hello, everyone! I'm Jishan Bhattacharya, and I had the incredible opportunity to intern at Qdrant this summer as part of the Qdrant Summer of Code 2024. Under the mentorship of [Andrey Vasnetsov](https://www.linkedin.com/in/andrey-vasnetsov-75268897/), I dived into the world of performance optimization, focusing on enhancing vector visualization using WebAssembly (WASM). In this article, I'll share the insights, challenges, and accomplishments from my journey — one filled with learning, experimentation, and plenty of coding adventures.
```

Please don't

**Pull request merged**